BPA/Ch.06/Mt.stan

```diff
@@ -49,7 +49,7 @@ generated quantities {
   int<lower=C> N;
   real pr;        // prob never captured given present
   real omega_nd;  // prob present given never captured; same for all animals
-  
+
   pr = prod(rep_vector(1.0, T) - p);
```

Comment:
This can just be `prod(1 - p)` if `p` is a vector---scalars in vector addition and subtraction get broadcast and the result is more efficient and also easier to read.